packages/table/src/headers/rowHeaderCell.tsx

```diff
@@ -56,21 +68,25 @@ export class RowHeaderCell extends React.Component<IRowHeaderCellProps, IRowHead
     };
 
     public render() {
-        const { className, isActive, isRowSelected, name, resizeHandle, style } = this.props;
-        const classes = classNames(className, "bp-table-header", {
+        const { className, isActive, isRowSelected, loading, name, resizeHandle, style } = this.props;
+        const rowHeaderClasses = classNames(className, "bp-table-header", {
+            [Classes.LOADING]: loading,
             "bp-table-header-active": isActive || this.state.isActive,
             "bp-table-header-selected": isRowSelected,
         });
+        const rowNameClasses = classNames("bp-table-row-name", { [Classes.LOADING]: loading });
```

Comment:
i put loading class on the header cell so that i can conditionally modify the styles of the row name child element. the headers are a bit complicated in that unlike cells, they hold a name and potentially other arbitrary content. the additional loading class on the row name child element itself is to prevent touching the styles of the other content